getting-started/deployment_guide.md

```diff
@@ -6,29 +6,22 @@ Check the [KubeArmor support matrix](support_matrix.md) to verify if your platfo
 
 ## Install KubeArmor
 ```
-curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
-karmor install
+helm repo add kubearmor https://kubearmor.github.io/charts
+helm repo update kubearmor
+helm upgrade --install kubearmor-operator kubearmor/kubearmor-operator -n kubearmor --create-namespace
+kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/pkg/KubeArmorOperator/config/samples/sample-config.yml
```

Comment:
Can you please include a statement indicating that "to install KubeArmor with default configurations (refer sample-config.yml)" before the `kubectl apply` command?

Reply:
Already referenced the doc in the next line for configuration and helm values, would that be enough?